presto-tests/src/test/java/com/facebook/presto/execution/TestHistoryBasedStatsTracking.java

```diff
@@ -361,4 +374,17 @@ private static Session createSession()
                 .setSchema("tiny")
                 .build();
     }
+
+    private static Session createSession(Integer historiesLimitingPlanNodeLimit)
+    {
+        return testSessionBuilder()
+                .setSystemProperty(USE_HISTORY_BASED_PLAN_STATISTICS, "true")
+                .setSystemProperty(TRACK_HISTORY_BASED_PLAN_STATISTICS, "true")
+                .setSystemProperty(USE_PERFECTLY_CONSISTENT_HISTORIES, "true")
+                .setSystemProperty(HISTORIES_LIMITING_PLAN_NODE_LIMIT, historiesLimitingPlanNodeLimit.toString())
```

Comment:
instead of creating this new function which duplicates a lot from the function above, can you inline in the test

```
Session session = testSessionBuilder(createSession())
      .setSystemProperty(HISTORIES_LIMITING_PLAN_NODE_LIMIT, 1)
```